src/flows/CostCalculator/Results/CostCalculatorResults.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { lazy, useState } from 'react';
+import { Euro } from 'lucide-react';
+
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/src/components/ui/card';
+import { Tabs, TabsList, TabsTrigger } from '@/src/components/ui/tabs';
+import { Badge } from '@/src/components/ui/badge';
+import { Separator } from '@/src/components/ui/separator';
+import { cn, formatCurrency } from '@/src/lib/utils';
+import { CostCalculatorBenefitsBreakdown } from './CostCalculatorBenefitsBreakdown';
+import { CostCalculatorContributionsBreakdown } from './CostCalculatorContributionsBreakdown';
+import { CostCalculatorEstimateResponse } from '@/src/client';
+
+const CostCalculatorResultsChart = lazy(
```

Comment:
Making charts visualization optional. As a consequence we lazy load CostCalculatorResultsChart  because of `recharts` which is quite large dependency.